db/schema.rb

```diff
@@ -330,7 +329,7 @@
     t.jsonb    "preferences",         default: {}
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "roles",               default: [], null: false, array: true
+    t.jsonb    "activity_count",      default: {}
```

Comment:
As far as I know, it's only used for the ribbon on https://www.zooniverse.org/projects/current. Some projects will have multiple workflow counts like Galaxy Zoo, but I don't think they're ever shown separately inside of zoo-home.